scheduler/cloudformation/template.go

```diff
@@ -305,15 +313,21 @@ func (t *EmpireTemplate) Build(app *scheduler.App) (interface{}, error) {
 			serviceProperties["Role"] = t.ServiceRole
 		}
 		resources[service] = map[string]interface{}{
-			"Type": "AWS::ECS::Service",
-			"Metadata": &serviceMetadata{
-				Name: p.Type,
-			},
+			"Type":       "AWS::ECS::Service",
 			"Properties": serviceProperties,
 		}
 
 	}
 
+	outputs[servicesOutput] = map[string]interface{}{
```

Comment:
This looks like it's the only output we use - are there any others that might be useful as well?  Maybe some that have the app name, the elb ID, etc?